lib/utils.js

```diff
@@ -671,6 +671,12 @@ exports.explain404 = function (str) {
   }
 };
 
+exports.info = function (str) {
+  if (process.browser && 'console' in global && 'info' in console) {
```

Comment:
this is probably not safe for node-webkit. you could do `typeof console !=== 'undefined'` instead of `'console' in global`